app/javascript/components/auth-key-pair-cloud/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+
+import MiqFormRenderer from '../../forms/data-driven-form';
+import createSchema from './auth-key-pair-cloud-form.schema';
+import miqRedirectBack from '../../helpers/miq-redirect-back';
+import { API } from '../../http_api';
+
+
+const AuthKeypairCloudForm = () => {
+  const [fields] = useState([]);
+  //   const cancelUrl = `/auth_key_pair_cloud/create/new?button=cancel`;
+  //   const SaveUrl = `/auth_key_pair_cloud/create/new?button=save`;
+
+  const onSubmit = (values) => {
+    miqSparkleOn();
+    const request = API.post('/api/auth_key_pairs', values);
+    request.then(() => {
+      const message = sprintf(
+
```

Comment:
You don't need this empty line and if the linter isn't complaining, I'd just make the whole expression a one-liner.